quickboosters/api/chat/dev/sample_data.py

```diff
@@ -0,0 +1,15 @@
+from flask_login import UserMixin
+from flask_sqlalchemy import SQLAlchemy
+from flask_admin import Admin
+from quickboosters.api.users.models import User
+from quickboosters.api.chat import models, ChatLog
+
+import datetime
+from quickboosters import db
+
+
+def Chatlog():
+    Chatlog = ChatLog(message='message',userfrom='user.username',created_date=datetime.datetime.now())
```

Comment:
variable name can't be the same as the class name of "ChatLog". You're importing "ChatLog" from models. You can't set an instance of the class to the same variable name.